index.html

```diff
@@ -1,31 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
+
 <body>
-  Instagram
+  <section class="first_section">
+
+  </section>
+
+  <main>
+
+    <section class="second_section">
+
+      <div >
+        <img src="images/phones.png" alt="">
+
+      </div>
+
+
+      <div class="right_container">
+        <div class="instagram_login">
+          <img src="images/brand.png" alt="">
+          <div class="button">
+            <img class="house_button" src="images/home.png" alt="home">
+            <span>Log in</span>
+          </div>
+
+
+        </div>
 
-  Capture and Share the World's Moments
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+        <div>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+          <article id="capture">
+            <!-- va a contener capture and share -->
+            <h2>Capture and Share <br> the World's Moments</h2>
 
-  Oh yeah, did we mention it's free?
+            <p>Instagram is a <strong>fast, beautiful </strong>and<strong> fun</strong> fun way to share your life with friends and family.</p>
```

Comment:
Try not to use strong that makes the style from the html and higher the semantic of the world when it's not needed here, better habit:
<span class='bold'> fun </span>
.bold {
    font-weight: bold;
}